packages/generate/src/queries.ts

```diff
@@ -38,6 +45,21 @@ currently supported.`);
 
   console.log(`Detected schema directory: ${params.schemaDir}`);
   const matches = await getMatches(root, params.schemaDir);
+
+  const errs:string[] = [];
+  function printSummary(){
+    const passedCount = matches.length - errs.length
```

Comment:
We prefer a more functional style in general. So instead of this function being a closure within this other function, it could be a top level function that receives `matches` and `errs` as arguments and either prints directly or returns a string that we `console.log` at the end.

Reply:
I was literally doing that the first time, closures are embedded in my muscle memory, sorry :)

Reply:
I'd still like to see this moved out into the top scope of this module (right after `formatError`) and have it take the `matches` and `errs` as arguments.